packages/oc-external-dependencies-handler/README.md

```diff
@@ -0,0 +1 @@
+# oc-external-dependencies-handler
```

Comment:
Can we add just a little description of what this is. Same for all the other modules. At least one single sentence that matches package's "description" field.